net-p2p/classified-ads/classified-ads-0.10.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PLOCALES="en fi sv da uk"
+PLOCALE_BACKUP="en"
```

Comment:
It does appear this eclass is not inherited.  PLOCALES && PLOCALE_BACKUP are indeed vars from the l10n eclass.